RecursiveExtractor.Cli/RecursiveExtractorClient.cs

```diff
@@ -0,0 +1,53 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.CST.RecursiveExtractor.Cli
+{
+    public class RecursiveExtractorClient
+    {
+		public static int Main(string[] args)
+		{
+
+			return CommandLine.Parser.Default.ParseArguments<ExtractCommandOptions>(args)
+			  .MapResult(
+				(ExtractCommandOptions opts) => Extract(opts),
+				errs => 1);
+		}
+
+		public static int Extract(ExtractCommandOptions options)
+        {
+			var extractor = new Extractor();
+			var extractorOptions = new ExtractorOptions()
+			{
+				ExtractSelfOnFail = true,
+				Parallel = true,
+			};
+			//if (options.Passwords.Any())
+   //         {
```

Comment:
suggest removing commented out code

Reply:
Can now complete this feature in this PR now that #20  is merged.  That will resolve #25 too.